utils/oscap-tool.h

```diff
@@ -118,6 +118,7 @@ struct oscap_action {
 	char *f_verbose_log;
 	/* others */
         char *profile;
+        const char *rule;
```

Comment:
@matusmarhefka Yes, it should be tabs. We follow the Linux Kernel coding style. https://www.kernel.org/doc/html/v4.10/process/coding-style.html

Unfortunately, some old code doesn't follow the coding style and uses spaces. In new pull request we use tabs, so that's why there is a mix. We haven't replaced spaces by tabs to avoid merge conflicts and possible problems with downstream patches.